packages/auth-browser/src/lib/chains/eth.ts

```diff
@@ -661,7 +662,13 @@ export const checkAndSignEVMAuthMessage = async ({
 
   // -- 9. finally, if the authSig is expired, re-sign
   // if it's not expired, then we don't need to resign
-  if (isSignedMessageExpired(authSig.signedMessage)) {
+  const checkAuthSig = validateSessionSig(authSig);
+
+  if (isSignedMessageExpired(authSig.signedMessage) || !checkAuthSig.isValid) {
+    if (!checkAuthSig.isValid) {
+      log(`Invalid AuthSig: ${checkAuthSig.errors.join(', ')}`);
```

Comment:
nit: should we use `logError` over `log`

Reply:
need to follow up - it's in auth-browser package, we haven't brought in `logError` over there yet

Reply:
Both are exported from `misc` and we already import it for `log`, so I think its already brought in.